stdlib/public/core/BridgeStorage.swift

```diff
@@ -31,9 +31,15 @@ internal struct _BridgeStorage<NativeClass: AnyObject> {
 
   // rawValue is passed inout to _isUnique.  Although its value
   // is unchanged, it must appear mutable to the optimizer.
+#if !$Embedded
```

Comment:
I think instead of cutting out specific parts of this, we should instead `#if` out the entire `_BridgeStorage` type -- there is no ObjC runtime in embedded Swift, and this type is only ever used to facilitate verbatim bridging.